addons/hr_expense/models/hr_expense.py

```diff
@@ -681,7 +694,8 @@ def _get_responsible_for_approval(self):
         return self.env.user
 
     def activity_update(self):
-        for expense_report in self.filtered(lambda hol: hol.state == 'submit'):
+        # TODO: See with TDE for 'submit' state
```

Comment:
It seems to me the expense manager has to maange the expense once it is submitted. Going from draft to submitted is the user role, am I wrong ?